### ISSUE TITLE
The specified procedure could not be found on Windows 10

### ISSUE THREAD

**Issue by wanglin86769** [Koromix/koffi#109]
Hello,
I want to use the dynamic link library **cas.dll** which depends on **ca.dll, Com.dll and gdd.dll**.

If I put the four dll files in the same directory as the Node.js file main.js, it works normally. However, if dll files are in a subdirectory **libs**, cas.dll cannot be loaded successfully with message "The specified module could not be found" and it looks like the dependent dlls cannot be located.

The following code produces error message "Error: Failed to load shared library: The specified **module** could not be found."

```
const koffi = require('koffi');
let LIBPCAS_PATH = '.\\libs\\cas.dll';
const libpcas = koffi.load(LIBPCAS_PATH);
```

If I add the subdirectory **libs** to system **PATH**,   the error message becomes "The specified procedure could not be found".

The following code produces error message "Error: Failed to load shared library: The specified **procedure** could not be found."

```
const koffi = require('koffi');
let LIBPCAS_PATH = '.\\libs\\cas.dll';
process.env.PATH += '.\\libs';
const libpcas = koffi.load(LIBPCAS_PATH);
```

How can I solve this problem?

**Comment by Koromix** [comment]
I've just published Koffi 2.6.8, which uses the `LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR` flag on Windows to search dependencies in the DLL directory.

Could you try it?

**Comment by wanglin86769** [comment]
Version 2.6.8 works, thanks for publishing the new version.
